README.md

```diff
@@ -14,7 +14,9 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
 
 🚫Add contributor info below, make sure add images and edit the social links for each member. Add to or delete these place-holders as needed
 
-|                                                      [Student 1](https://github.com/)                                                       |                                                       [Student 2](https://github.com/)                                                        |                                                      [Student 3](https://github.com/)                                                       |                                                       [Student 4](https://github.com/)                                                        |                                                      [Student 5](https://github.com/)                                                       |
+|                                                      [Jose Sustaita](https://github.com/JoseSustaita)                                                       |                                                       [Emilio Dellrodar](https://github.com/Dellrodar)                                                        |                                                      [Matthew Taylor](https://github.com/MTaylor-tech)                                                       |                                                       [Idong Essien](https://github.com/idongessien)                                                        |                                                      [Devin Ong](https://github.com/hoangvu71)
```

Comment:
Since team composition has changed just update this to correctly reflect the team and it should be good to go!